beefy-gadget/src/lib.rs

```diff
@@ -44,10 +44,14 @@ use sp_runtime::{
 	traits::{Block as BlockT, Hash as HashT, Header as HeaderT, NumberFor, Zero},
 };
 
+mod error;
+
 pub mod notification;
 
 use notification::BeefySignedCommitmentSender;
 
+pub use error::Error;
```

Comment:
Shall we also export `Result`?  I think this is a common practice to use `my_lib::Result<()>` instead of `Result<(), my_lib::Error>`.
Note that I'm very much against importing `my_lib::Result` to the scope and overriding `std::result::Result` though (i.e. only full path usage).

Reply:
I made the error type `pub(crate)` for the time being, since currently it is really only used for `beefy_gadget` internal error handling.